org.eclipse.ice.bats/src/main/docker/Dockerfile.fuseki

```diff
@@ -0,0 +1,22 @@
+FROM openjdk:11.0.1-jre
```

Comment:
I think this is another reason we should set the build target to 11: the Docker container is using it.

Reply:
Agreed!